lib/db_utils.pm

```diff
@@ -202,42 +194,41 @@ sub check_postgres_db {
     my $image = shift;
     my $db_ip = get_var('POSTGRES_IP');
```

Comment:
let's make in required as above and we can remove the L#206

Reply:
Careful, then it becomes a real requirement for the test run, and this alters the logic!

I made it required above, because the call of `push_image_data_to_db` really requires it. In `check_postgres_db` the logic is to only print a warning, if the host is not defined.